### DISCUSSION TITLE
Leaving the "External" Adam image SD card in whilst another "Internal" SD card is inserted.

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#157] [SD topics]
Just wondered...I haven't yet figured how to get Quake 2 and some other FPS games running in the Adam image. Almost certainly it is simple to do, I just haven't put the time in to finding out which file I need to put where.

However, it is working fine as part of the CBMT 5 internal SD card image. I found almost none of the games on the external SD card image for the CBMT 5 firmware worked. Then I tried the Adam image and all of my dreams came true. However, until I work out how to get the few ports that came on the CBMT 5 internal image, running with Adam...IF I just swap out the Adam internal card with the CBMT one, can I leave the Adam external card in. Of course the CBMT internal card wont read the file system...but I just wondered if the Adam external card could get damaged or corrupted if left in when a different custom firmware internal card is put in?

**Comment by eduardofilo** [comment]
Quake2 needs a new core (vitaquake2) available in the lasts RA distributions. I haven't find documentation yet, only a [post](https://www.libretro.com/index.php/vitaquake-ii-libretro-core-wip-available-soon-high-refresh-rates-in-libretro-core-games/). The core_info files don't offer too much info also:

>description = "A port of the VitaQuake 2 source port of iD's Quake 2 engine to libretro. There is a separate core for each of the Quake 2 mission packs, 'Rogue', 'Zaero' and 'Xatrix'. This core is for the main game. This core loads games in the *.pak format."
>description = "A port of the VitaQuake 2 source port of iD's Quake 2 engine to libretro. There is a separate core for each of the Quake 2 mission packs, 'Rogue', 'Zaero' and 'Xatrix'. This core is for the 'Ground Zero' mission pack. This core loads games in the *.pak format."

I don't know about the CBMT image, but I doubt it will delete files on the external card. I guess it won't create them either. Adam creates some directories, but not files, so it is unlikely that both images will interfere with each other.

**Reply by chrisstonehill** [reply]
Thanks Eduardo. Maybe I'll try and get a bit further this evening. I know on the Retro Game Corps website there is info about Doom and Quake on to the RG350 devices...but that info is only for the stock firmware...perhaps doesn't apply to how the Adam image does things?

**Reply by eduardofilo** [reply]
Maybe, although 80% of the software designed for stock works well in ODb.
